### ISSUE TITLE
bug when moving a rule into a group

### ISSUE THREAD

**Issue by Horzo** [mistic100/jQuery-QueryBuilder#316]
Hi,

I'm getting a crash when trying to move a rule into an empty group.

the hierarchy is like this:
-grp0
---rule0
---grp1 (empty)
---rule1 (try to move this one to grp1)

I'm using IE11

hope this helps reproducing the bug.

**Comment by Horzo** [comment]
a temporary fix:
replacing index===0 by index<=0

> 
'move':function(e,node,group,index) {
                    node.$el.detach();
                    --if(index===0) {
                    if(index<=0) {
                        node.$el.prependTo(group.$el.find(">"+selectors.rs_lst));
                    }
                    else {
                        node.$el.insertAfter(group.rules[index-1].$el);
                    }
                    self.refreshAllGroupConditions();
                }

**Comment by mistic100** [comment]
Which version ? Please precise what is "a crash"

**Comment by mistic100** [comment]
Ok it's not a "crash" at all, just an error in the console.
